### DISCUSSION TITLE
Bug Report: FlxBar: max cannot be less than or equal to min, im new to coding and don't know how to fix this

### DISCUSSION THREAD

**Discussion by Cuph3ad-tech** [KadeArchive/Kade-Engine#423] [Q&A] [answered]
kade engine 1.4.2 version. windows 10 x64, i have started learning to code kade engine recently, and i found this issue, is there any fix?

**Comment by theDetourist** [comment, marked as answer]
Does that happen when you play MILF or the Tutorial? If so, update to the [latest nightly version](https://ci.appveyor.com/project/KadeDev/kade-engine-windows/build/artifacts).

**Reply by Cuph3ad-tech** [reply]
it was problem with audio file, i fixed it now, thanks for sugggestion anyway

**Reply by Burger-Gaming** [reply]
Can you explain what the problem was? I'm having this issue as well.

**Reply by willboyz92nd** [reply]
wait whats the fix i have been having the same error

**Reply by Burger-Gaming** [reply]
Found the issue, you just had to click that the song needs voices.

**Reply by Burger-Gaming** [reply]
nevermind, i didnt solve it

**Reply by willboyz92nd** [reply]
for everyone having this issue i found the fix when you use a song make sure you use a converter tool to convert mp3 to ogg just renaming the file to .ogg will give you the error

**Reply by myst-rblx** [reply]
> for everyone having this issue i found the fix when you use a song make sure you use a converter tool to convert mp3 to ogg just renaming the file to .ogg will give you the error

not working
